packages/editor/src/lib/app/types/emit-types.ts

```diff
@@ -0,0 +1,90 @@
+import { TLPageId, TLRecord } from '@tldraw/tlschema'
+import { TLChange } from '../App'
+import { TLEventInfo } from './event-types'
+
+/** @public */
+export interface TLEventMap {
```

Comment:
These are the types that we allow to be emitted by `app.emit`. They used to be untyped, now they're typed!